sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj

```diff
@@ -1,13 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
     <LangVersion>8.0</LangVersion>
     <AssemblyName>SonarAnalyzer.CSharp</AssemblyName>
     <RootNamespace>SonarAnalyzer</RootNamespace>
-    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
-    <RestoreLockedMode>true</RestoreLockedMode>
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>

```

Comment:
another nitpick observation is that we're now inconsistent where we put the TargetFrameworks - here it's at the end of the PropertyGroup, in the other csproj (like CFG or common, which haven't changed) it's at the beginning

I don't care, TBH, but it's just a comment. I don't mind leaving it as it is.

Reply:
I've used the project config UI from Rider. Not sure it matters that much.